roles/ceph-nfs/templates/ganesha.conf.j2

```diff
@@ -12,15 +12,17 @@ EXPORT
 
 	Access_Type = {{ ceph_nfs_ceph_access_type }};
 
-	NFS_Protocols = {{ ceph_nfs_ceph_protocols }};
+	Protocols = {{ ceph_nfs_ceph_protocols }};
 
-	Transport_Protocols = TCP;
+	Transports = TCP;
 
 	Sectype = sys,krb5,krb5i,krb5p;
 
 	FSAL {
 		Name = CEPH;
 	}
+
+        {{ ganesha_ceph_export_overrides }}
```

Comment:
@leseb is it ok that these variables are empty and undeclared? in defaults/main.yml they are declared like this:
`#ganesha_ceph_export_overrides:`